preptime/routes/news.js

```diff
@@ -1,12 +1,25 @@
+const { default: axios } = require("axios");
 const express = require("express");
 const router = express.Router();
 
 router.get("/", (req, res) => {
   res.send("NEWS API IS RUNNING");
 });
 
-router.get("/latest", (req, res) => {
-  res.send("Latest news - random word: bicycle");
+router.get("/source/:source", async (req, res) => {
+  const { source } = req.params;
+  const response = await axios.get(
+    `https://footballnewsapi.netlify.app/.netlify/functions/api/news/${source}`
+  );
+  res.send(response.data);
+});
+
+router.get("source/:source", async (req, res) => {
+  const { source } = req.params;
+  const response = await axios.get(
+    `https://footballnewsapi.netlify.app/.netlify/functions/api/news/fourfourtwo/${source}`
```

Comment:
maybe put the main url into a `const` and add the `fourfourtwo` to it so that it's reusable?